litestar/_signature/models/base.py

```diff
@@ -52,9 +56,7 @@ def _create_exception(cls, connection: ASGIConnection, messages: list[ErrorMessa
             if ("key" in err_message and err_message["key"] not in cls.dependency_name_set) or "key" not in err_message
         ]:
             return ValidationException(detail=f"Validation failed for {method} {connection.url}", extra=client_errors)
-        return InternalServerException(
-            detail=f"A dependency failed validation for {method} {connection.url}", extra=messages
-        )
+        return InternalServerException
```

Comment:
Similar to #1857 - we shouldn't be leaking any implementation detail from ISEs.

This should be in a separate PR as we want this weather we get this one merged or not.

Reply:
But this is not the place to do this. We should give access to this when `debug=true`

Reply:
Oh, I see - I'll back this out then.